src/vulkan/pipeline.cc

```diff
@@ -299,11 +307,16 @@ Result Pipeline::SendDescriptorDataToDeviceIfNeeded() {
   if (!r.IsSuccess())
     return r;
 
-  for (const auto& desc : descriptors_) {
-    r = desc->CreateOrResizeIfNeeded(command_->GetCommandBuffer(),
-                                     memory_properties_);
-    if (!r.IsSuccess())
-      return r;
+  for (auto& info : descriptor_set_info_) {
+    if (info.empty)
```

Comment:
This check isn't needed because the for() below won't do anything if it's empty

Reply:
removed